WordPressAuthenticator/Authenticator/WordPressAuthenticatorConfiguration.swift

```diff
@@ -47,6 +47,14 @@ public struct WordPressAuthenticatorConfiguration {
     ///
     let googleLoginScheme: String
 
+    internal var googleClientId: GoogleClientId {
+        guard let clientId = GoogleClientId(string: googleLoginClientId) else {
+            fatalError("Could not init GoogleClientId from developer provided value.")
+        }
+
+        return clientId
+    }
```

Comment:
This way, we won't need instantiate a `GoogleClientId` when passing the configuration options to the new SDK-less flow. It will keep the code more concise.

Reply:
I'm not sure why it's explicitly `internal`, given that's the default...